data/images/readme.md

```diff
@@ -0,0 +1,5 @@
+#### To make predictions of new images, place them in this folder.
+
+The expected format for images is .jpg
```

Comment:
Maybe mentioning requested image size here + linking this `readme.md` file from the main one would be helpful.